packages/scaffold/src/views/w3m-compatible-networks-view/index.ts

```diff
@@ -0,0 +1,52 @@
+import { AssetUtil, CoreHelperUtil, NetworkController } from '@web3modal/core'
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { ifDefined } from 'lit/directives/if-defined.js'
+import styles from './styles.js'
+
+@customElement('w3m-compatible-networks-view')
+export class W3mCompatibleNetworksView extends LitElement {
+  public static override styles = styles
+
+  // -- Render -------------------------------------------- //
+  public override render() {
+    return html` <wui-flex
+      flexDirection="column"
+      .padding=${['xs', 's', 'm', 's'] as const}
+      gap="xs"
+    >
+      <wui-banner
+        text="Only send funds from these networks to be able receive them"
```

Comment:
Is this copy correct @Sam-Newman ?

Reply:
No, it doesn't make sense, suggestion:

"You can only receive assets on these networks"

@svenvoskamp @tomiir